byterun/minor_gc.c

```diff
@@ -170,6 +176,56 @@ void caml_set_minor_heap_size (asize_t bsz)
   reset_table ((struct generic_table *) &caml_custom_table);
 }
 
+mlsize_t caml_distance_to_closure_environment(value closure)
+{
+#ifndef NATIVE_CODE_AND_NO_NAKED_POINTERS
+  return 0;
```

Comment:
This function seems to be correct without NO_NAKED_POINTERS.
The ifndef should only be on NATIVE_CODE even if it is unused